packages/analytics-client-common/src/attribution/web-attribution.ts

```diff
@@ -23,13 +26,16 @@ export class WebAttribution {
     this.storage = config.cookieStorage as unknown as Storage<Campaign>;
     this.storageKey = getStorageKey(config.apiKey, 'MKTG');
     this.currentCampaign = BASE_CAMPAIGN;
+    this.sessionTimeout = config.sessionTimeout;
```

Comment:
Should we keep a reference to the config instead of the actual values? What happens when the last event time changed and we call init once again mid session?

Reply:
We call the init in the mid session, then we will get the latest lastEventTime? We are updating the this.config.lastEventTime while sending out the event.